shared/chat/conversation/info-panel/index.js

```diff
@@ -147,11 +89,57 @@ type ParticipantRow = {
   key: string,
 }
 
-type BigTeamRow = HeaderRow | ParticipantRow
-// For virtualizing big team participants list
-const _renderBigTeamRow = (i: number, props: BigTeamRow) => {
+type TeamRow = ConversationFooterRow | SmallHeaderRow | BigHeaderRow | ParticipantRow
+type RowType = $PropertyType<TeamRow, 'type'>
+
+const _renderTeamRow = (i: number, props: TeamRow) => {
   switch (props.type) {
-    case 'header':
+    case 'conversation footer':
+      return (
+        <Box key={props.key} style={{...globalStyles.flexBoxColumn, alignItems: 'stretch'}}>
+          <Divider style={{marginBottom: 20, marginTop: 10}} />
+
```

Comment:
lets get rid of these extra spaces, we don't have this elsewhere

Reply:
Done (had to set the small property on the button bar, too)